pkg/platform/aws/aws.go

```diff
@@ -78,6 +79,7 @@ type config struct {
 	CertsValidityPeriodHours int               `hcl:"certs_validity_period_hours,optional"`
 	WorkerPools              []workerPool      `hcl:"worker_pool,block"`
 	DisableSelfHostedKubelet bool              `hcl:"disable_self_hosted_kubelet,optional"`
+	Oidc                     *oidc.Config      `hcl:"oidc,block"`
```

Comment:
Should `Oidc` be capitalized?
```suggestion
	OIDC                     *oidc.Config      `hcl:"oidc,block"`
```

Reply:
I think capitalization doesn't look visually appealing for more than 3 letters.

Reply:
updated to OIDC